CorsixTH/Lua/game_ui.lua

```diff
@@ -96,6 +98,15 @@ function GameUI:GameUI(app, local_hospital, map_editor)
   self.shake_screen_intensity = 0
 
   self.announcer = Announcer(app)
+  
+  
+  -- Set the scrolling keys.
+  self.scroll_keys = {
+    [tostring(app.hotkeys["scroll_up"])]   = {x =   0, y = -10},
```

Comment:
I don't mind if you remove the extra spaces, or line up the table like the old scroll_keys.